uktrade/data-hub-frontend
# Bugfix/amend opportunity pagination

## Description of change

This PR adds a few finishing touches to the first implementation of the investments -  UK opportunities tab. This includes fixing a bug with the pagination and updating the functional tests for the tab. It also includes the amendment of the casing of UK opportunities in the Tab and the add entity button.

## Test instructions

Head to the investments page in the main nav, then click on the opportunities tab. You should see that the casing for opportunities in the tab label is lower case. You should also see that the button to add another opportunity should also have a lower case 'o'. Head to the bottom of the page, the pagination should work, and there should only be 10 entries to a page. 


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
